pages/article/[articleId].js

```
@@ -0,0 +1,24 @@
+export async function getStaticProps({ params }) {
+  const article = articles.find(a => a.id === params.articleId);
+  const category = categories.find(cat => cat.id === article.category);
+  return { props: { article, category } };
+}
+
+export default function ArticlePage({ article, category }) {
+  return (
+    <div className={styles.container}>
+      <Link href={`/category/${category.id}`} className={styles.backLink}>← {category.title}</Link>
+      <h1 className={styles.title}>{article.title}</h1>
+      <div className={styles.content} dangerouslySetInnerHTML={{ __html: article.content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)